### DISCUSSION TITLE
Title Change, Adding other Pages

### DISCUSSION THREAD

**Discussion by indangasy** [mediacms-io/mediacms#735] [Q&A]
I trust you are well and you gonna assist me on this minor issue(s)

1) How can i edit the logo size? i have managed to replace it. Which file should i edit, the path?
2) Would like to edit the footer. Possible? Add something where it says Powered by mediacms.io
3) I have edited the file to change the website title, still shows MediaCMS. Same with about us! Which files do i need to edit to effect the changes?
4) How do Add other pages / links on the left menu

**Comment by anjanesh** [comment]
I would like to know how to edit the About page as well.

**Comment by touni1** [comment]
I’m joining the question — have you managed to solve it?
